pyproject.toml

```diff
@@ -11,5 +11,58 @@ exclude = ["venv"]
 "avro/resources/dictionary.py" = ["F601"]
 "tests/test_main.py" = ["F601"]
 
-[tool.ruff.format]
-quote-style = "single"
\ No newline at end of file
+[tool.poetry]
+name = "avro-py"
+version = "2024.5.1"
+description = "A modern Pythonic implementation of Avro Phonetic."
+authors = ["HitBlast <hitblastlive@gmail.com>"]
+license = "MIT"
+readme = "README.md"
+packages = [{include = "avro"}]
+homepage = "https://pypi.org/project/avro-py"
+repository = "https://github.com/hitblast/avro.py"
+documentation = "https://hitblast.github.io/avro.py"
```

Comment:
This likely wont work.

Check : https://github.com/baseplate-admin/django-ltree-2/blob/55584e9075cad1c9f4a0141523048630a52a4121/pyproject.toml#L22-L25

Reply:
https://python-poetry.org/docs/pyproject/

Here, you can have a look at the pyproject.toml configuration guideline I suppose.

Reply:
Hmm so basically this could be a new feature. 

I always used this : https://python-poetry.org/docs/pyproject/#urls

Because you can add custom urls like
```
"Bug Tracker" = "https://github.com/hitblast/avro.py/issues"
```

Reply:
I see. Might add them if necessary later.